explainer.md

```diff
@@ -141,6 +141,22 @@ This proposal restricts what would be possible with `launch` events, and could a
 - If an existing window is open and the app wants to open a new window, a flicker of the existing window will be shown as it is focused prior triggering the event
 - In terms of ergonomics, developers must be careful to register `launch` event handlers on all windows and to co-ordinate between them. Often this will involve going through the service worker anyway.
 
+#### How well will this support behavior of existing native applications?
+
+Below is a not-at-all scientific collection of how a few common apps handle files being opened. SW Launch refers to the case where we fire a launch event on the service worker, while client launch refers to a theoretical event handler on the client window.
```

Comment:
nit: perhaps reword to flow from the above?
The table below summarizes the behaviors of existing native apps and whether they could be reproduced if the launch event the fires in the service worker or the window.